Reference_Manual.md

```diff
@@ -1202,114 +1209,111 @@ You can also fine-tune this feature using `client_config::set_protperty` API as
 
-#### 6.1.1. TLS/SSL for Hazelcast Members
+### 6.1.2. TLS/SSL for Hazelcast C++ Client
 
-Hazelcast allows you to encrypt socket level communication between Hazelcast members and between Hazelcast clients and members, for end to end encryption. To use it, see the [TLS/SSL for Hazelcast Members section](http://docs.hazelcast.org/docs/latest/manual/html-single/index.html#tls-ssl-for-hazelcast-members) in the Hazelcast IMDG Reference Manual.
+The Hazelcast C++ client uses [Boost Asio](https://www.boost.org/doc/libs/1_75_0/doc/html/boost_asio.html) library for networking and secure communication. 
 
-#### 6.1.2. TLS/SSL for Hazelcast C++ Clients
+To use TLS/SSL with your Hazelcast C++ client, you should build the library with OpenSSL feature turned on. By default, this feature is not turned on. Here are different ways to install the library with SSL support:
 
-To use TLS/SSL with your Hazelcast C++ client, you should perform the following:
+* CMake Users: Provide flag `-DWITH_OPENSSL=ON` flag when configuring. Please note that the CMake `find_package` should be able to locate the OpenSSL installation, otherwise the cmake config will fail. The rest is as usual [installation with cmake](#131-cmake-users).
+* Conan Users: Use option `with_openssl` while installing the conan project. An example command you can use in the build folder of your project with conan: `conan install -o hazelcast-cpp-client:with_openssl=True .. --build=hazelcast-cpp-client`. This command will install the `hazelcast-cpp-client` with the OpenSSL support.
 
-* Provide the compile flag **`-DHZ_BUILD_WITH_SSL`** when compiling, since the TLS feature depends on OpenSSL library.
-* Install the OpenSSL library to your development environment.
-* Enable the SSL in the client network configuration.
-* Specify the correct path to the CA verification file for the trusted server. This path can be relative to the executable working directory.
-* If needed, set the cipher suite to be used via the SSL configuration.
+Once the library is installed properly, you can enable the ssl feature in the client config.  
 
-You can set the protocol type. If not set, the configuration uses `tlsv12` (TLSv1.2) as the default protocol type and version..
 
-The following is an example configuration.
+You can set the protocol type. If not set, the configuration uses `tlsv12` (TLSv1.2) as the default protocol type and version.
+
+There may be different ways to configure ssl communication at the client side. We utilize the `boost::asio::ssl::context` for various configuration possibilities. You can set the verify mode to ignore or verify the server certificate, you can set the used ssl protocol, you can add verify callbacks, and such. The details can be found at the [Boost asio ssl context documentation](https://www.boost.org/doc/libs/1_75_0/doc/html/boost_asio/reference/ssl__context.html).
```

Comment:
... to configure the SSL communication ...

You can set the verify mode to ignore or verify the server certificate, set the used SSL protocol, add verify callbacks, and such.